dev/lib/product_taxonomy/models/extended_attribute.rb

```diff
@@ -14,6 +20,7 @@ class ExtendedAttribute < Attribute
     #   instead.
     def initialize(name:, handle:, description:, friendly_id:, values_from:)
       @values_from = values_from
+      values_from.add_extended_attribute(self) if values_from.is_a?(Attribute)
```

Comment:
I would have expected this to be `if values_from.is_a?(ExtendedAttribute)` based on other code